### PR TITLE
system/telemetry metrics OTLP GRPC Exporter: add CA certificate support

### PR DESCRIPTION
**Pull request by ms-jcorley**
#### Description
Adds support for CA certificates to the OTLP gRPC exporter for metrics in system/telemetry.
Currently if a "Certificate" is specified in the config, it is silently ignored.

#### Testing
Both a positive and negative unit test was added.
The good test CA certificate was copied from another unit test.
I'm going to file an issue for updating all test certificates as all the ones I checked seem to expire in 2032 which is only 8 years away.

#### Documentation
None.

### CODE REVIEW

File: service/telemetry/internal/otelinit/config.go
Comment (dmathieu) [inline service/telemetry/internal/otelinit/config.go:229]: Nit, but do we shouldn't need that extra indentation here

```suggestion
		} else if otlpConfig.Certificate != nil {
```

**Review by dmathieu — COMMENTED** [review]
This will need a changelog entry.

File: service/telemetry/internal/otelinit/config.go
Comment (ms-jcorley) [reply service/telemetry/internal/otelinit/config.go]: Ah yes, I was thinking about future code there and also didn't realize I should run make lint (I'm new here)... fixed.

**Comment by ms-jcorley** [comment]
> This will need a changelog entry.

@dmathieu, changelog was added and I fixed the lint issue.  Waiting for a review/approval.

**Comment by ms-jcorley** [comment]
> Can you run `make gotidy`?

@dmathieu, Done.

**Review by dmathieu — APPROVED** [review]

**Comment by ms-jcorley** [comment]
Hi @codeboten, it looks like this is blocked on your final review.  This is quite a small change, could you take a look when you get a chance?
Thank you!

**Review by codeboten — APPROVED** [review]
Thanks @ms-jcorley

**Pull request merged**
